### DISCUSSION TITLE
Not able to get css file when url set as target

### DISCUSSION THREAD

**Discussion by mubicr7** [chimurai/http-proxy-middleware#762] [Q&A]
Hello

I am proxying a call to an external webpage which has css and js files in the HREF tag. When set that href url as the target i dont get the css file instead i get the index page.

https://oecconsoles.pages.openearth.community/oec-doc/assets/stylesheets/main.2d9f7617.min.css
Opening this browser give the css file content.

But when i set this as target and call the proxies endpoint or page having the proxied endpoint as the href in the style tag return 404 not found.

**Comment by chimurai** [comment]
You can try [v3@beta](https://github.com/chimurai/http-proxy-middleware/discussions/768).

You should be able to get better debug and log data.

```js
createProxyMiddleware({
   logger: console
})
```

```shell
# enable debug logging
DEBUG=http-proxy-middleware* node app.js
```
